CS483/CS483/Kartaclysm/Components/ComponentRacer.h

```diff
@@ -36,9 +36,9 @@ namespace Kartaclysm
 		virtual void Update(const float p_fDelta) override;
 
 		HeatStroke::GameObject* GetKart() {return m_pKart;}
-		void SetKart(HeatStroke::GameObject* p_pKart);
+		void SetKart(HeatStroke::GameObject* p_pKart) { m_pKart = p_pKart; }
```

Comment:
They still are, it's just handled at the time of their creation in StateRacing::GenerateRacer. This allows the kids and grandkids to access the GUID in their Init() calls, which gives them the "PlayerX" string they need